api/RTCDTMFToneChangeEvent.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "RTCDTMFToneChangeEvent": {
```

Comment:
The member <code>tone</code> and the constructor <code>RTCDTMFToneChangeEvent</code> are missing (sub-features).

Reply:
Yeah removed those after the linter moaned about that file not requiring subfeatures. Today though, I understand what's going on a little deeper with the workflow. Will have another go.